build-tools/scripts/mono.mk

```diff
@@ -37,11 +37,14 @@ export MONO_OPTIONS
 endif   # $(MONO_OPTIONS) != ''
 
 ifeq ($(OS),Darwin)
-JI_MONO_FRAMEWORK_PATH = /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.1.dylib
-JI_MONO_INCLUDE_PATHS = /Library/Frameworks/Mono.framework/Headers/mono-2.0
-JI_MONO_LIBS = -L /Library/Frameworks/Mono.framework/Libraries -lmonosgen-2.0
+JI_MONO_BASE = /Library/Frameworks/Mono.framework/
+JI_MONO_PATH = $(JI_MONO_BASE)Libraries/
+JI_MONO_FRAMEWORK_PATH = $(JI_MONO_PATH)libmonosgen-2.0.1.dylib
+JI_MONO_INCLUDE_PATHS = $(JI_MONO_BASE)Headers/mono-2.0
+JI_MONO_LIBS = -L $(JI_MONO_PATH) -lmonosgen-2.0
 endif
 ifeq ($(OS),Linux)
+JI_MONO_PATH = $(dir `which mono`)/../lib/
```

Comment:
This should be named `$(JI_MONO_LIB_PATH)`; "mono path" implies the path for _mono_, which isn't the case.

`$(JI_MONO_LIB_PATH)` should also be documented at the top of the file, along with all the other values which are exported.

Reply:
OK, updated.